manifests/client/monitor.pp

```diff
@@ -12,34 +12,72 @@
 # @param files A hash of filenames as key and sources as values to add to the xymon files
 # @param sudo A sudo::conf hash with sudo definitions the xymon user should be allowed to use
 # @param packages A puppet package hash with packages that are required for the monitor to work
+# @param logrotate A hash containing definitions to configure logfile rotation
```

Comment:
Please include documentation about the available keys in the hash using @option docstrings (see https://puppet.com/docs/puppet/7.4/puppet_strings.html#available-strings-tags)

Reply:
@dploeger Can we discuss about this? A lot of the parameters are hashes and it wouldn't make sense to document it just for logrotate.
Either we do it for ALL hashes or we refer to the example. What do you think?

Reply:
I'd suggest doing it for ALL hashes. (If we simply create defined types with the hash we can simply point to the documentation of that type)

Reply:
Documented all now, which are not documented elsewhere already (like "packages" and "sudo:conf").